test/fake_modules/package_import_map/index.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
+
+// This file includes some experimental ES7 syntax enabled in Babel by default. Reference: https://babeljs.io/docs/usage/experimental/
```

Comment:
Is this true? Or a copy/paste?

Reply:
Ah definitely copy/paste, sorry

Reply:
No problem, I wasn't sure, haven't looked at javascript language changes in years :D

Reply:
I pushed a cleaned up version of this one